src/uct/ib/base/ib_md.c

```diff
@@ -185,6 +185,11 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Enable setting CPU affinity of memory registration threads.",
      ucs_offsetof(uct_ib_md_config_t, ext.mt_reg_bind), UCS_CONFIG_TYPE_BOOL},
 
+    {"PCI_RELAXED_ORDERING", "auto",
+     "Enable relaxed ordering for PCI writes from device to memory, "
```

Comment:
why only writes? aren't RO reads enabled too ?

Reply:
in case, I'd just drop "writes from device to memory" and mention "PCIe transactions"